app/index.js

```diff
@@ -426,7 +426,7 @@ function showThankYou() {
     //Find out how many seconds has passed to give response
     const endFeedback = new Date();
     const startFeedback = new Date(feedbackData['startFeedback']);
-    feedbackData['responseSpeed'] = (endFeedback - startFeedback) / 1000.0;
+    feedbackData['responseSpeed'] = ((endFeedback - startFeedback) / 1000.0).toFixed(1);
```

Comment:
I needed to add this since data in influx are stored as float and if you do not select any questions (default option) then the response speed is 0 (integer) and not accepted by influx.